rust-toolchain

```diff
@@ -0,0 +1 @@
+1.36.0
```

Comment:
There was discussion on this before, and the conclusion is we don't want a toolchain file since this is just a library, not a final binary

Reply:
update